force-app/main/default/lwc/accountModelSettings/accountModelSettings.js

```diff
@@ -192,15 +212,11 @@ export default class AccountModelSettings extends LightningElement {
         this.template.querySelector("c-settings-save-canvas").handleHierarchySettingsChange(hierarchySettingsChange);
     }
 
-    handleAdministrativeAccountNamingChange(event) {
-        // hide/show custom Admin Account naming input field
-        this.showCustomAdministrativeAccountNaming =
-            event.detail.value === this.labelReference.accountNamingComboboxCustomOption;
-
-        // add updated setting to hierarchySettingsChanges object
+    handleLeadNamingAccountChange(event) {
+        // add selected values to hierarchySettingsChanges object
         let hierarchySettingsChange = {
-            settingsType: "string",
-            settingsName: "Admin_Account_Naming_Format__c",
+            settingsType: "array",
+            settingsName: "Lead_Naming_Account_RTypes__c",
             settingsValue: event.detail.value,
         };
```

Comment:
Please restore the Administrative Account Naming Change method and have the lead naming account change as a separate method, as it seems was your original intent.

Reply:
Added handleAdministrativeAccountNamingChange back